app/scripts/modules/azure/serverGroup/configure/serverGroupBasicSettingsSelector.directive.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular
+  .module('spinnaker.azure.serverGroup.configure.azureServerGroupBasicSettingsSelector', [
+    require('../../../core/serverGroup/configure/common/basicSettingsMixin.controller.js'),
+    require('../../../core/region/regionSelectField.directive.js'),
+    require('../../../core/account/accountSelectField.directive.js'),
+    require('../../subnet/subnetSelectField.directive.js'),
+  ])
+  .directive('azureServerGroupBasicSettingsSelector', function() {
+    return {
+      restrict: 'E',
+      scope: {
+        command: '=',
+        application: '=',
+        hideClusterNamePreview: '=',
+      },
+      templateUrl: require('./serverGroupBasicSettingsSelector.directive.html'),
+      controller: 'ServerGroupBasicSettingsSelectorCtrl as basicSettingsCtrl',
+    };
+  })
+  .controller('ServerGroupBasicSettingsSelectorCtrl', function($scope, $controller, RxService, imageReader, namingService, $modalStack, $state) {
```

Comment:
probably want to prefix this controller name with `azure` - there is not another `ServerGroupBasicSettingsSelectorCtrl`, but the other providers (cf, gce, titan) have prefixed this name.